library/ssl_tls.c

```diff
@@ -1615,11 +1620,75 @@ int mbedtls_ssl_set_hs_ecjpake_password( mbedtls_ssl_context *ssl,
                                          const unsigned char *pw,
                                          size_t pw_len )
 {
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
+    psa_pake_cipher_suite_t cipher_suite = psa_pake_cipher_suite_init();
+    psa_key_attributes_t attributes = PSA_KEY_ATTRIBUTES_INIT;
+    psa_pake_role_t psa_role;
+    psa_status_t status;
```

Comment:
Minor: In a situation where the whole body (apart from one check) is different in a `USE_PSA_CRYPTO` vs non `USE_PSA_CRYPTO` build I would rather suggest having two separate function bodies (that's not worth waiting for the whole new CI run though).